xdsl/dialects/func.py

```diff
@@ -62,6 +64,42 @@ def get_result_types(cls, op: Operation) -> tuple[Attribute, ...]:
         return op.function_type.outputs.data
 
 
+class CallOpSymbolUserOpInterface(SymbolUserOpInterface):
+    def verify(self, op: Operation) -> None:
+        assert isinstance(op, CallOp)
+
+        found_callee = SymbolTable.lookup_symbol(op, op.callee)
+        if not found_callee:
+            raise VerifyException(f"'{op.callee}' could not be found in symbol table")
+
+        if not isinstance(found_callee, FuncOp):
+            raise VerifyException(f"'{op.callee}' does not reference a valid function")
+
+        if len(found_callee.function_type.inputs) != len(op.arguments):
+            raise VerifyException("incorrect number of operands for callee")
+
+        if len(found_callee.function_type.outputs) != len(op.result_types):
+            raise VerifyException("incorrect number of results for callee")
+
+        for idx, (found_oprnd, oprnd) in enumerate(
```

Comment:
Are we on a vowel budget? :)

Reply:
LOL, my inner C naming conventions came forth!
I'll amend.